apps/public/public.go

```diff
@@ -251,3 +251,11 @@ func (pca Client) AcquireTokenByAuthCode(ctx context.Context, scopes []string, o
 func (pca Client) Accounts() []Account {
 	return pca.base.Accounts()
 }
+
+// AcquireTokenInteractive acquires a security token from the authority using the default web browser to select the account.
+func (pca Client) AcquireTokenInteractive(ctx context.Context, scopes []string) (AuthResult, error) {
```

Comment:
We should have an end to end test for this.  But this interacts with the browser.

@abhidnya13 @keegan-caruso do you know how we are going to handle this?